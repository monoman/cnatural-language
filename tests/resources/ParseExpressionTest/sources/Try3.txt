() => {
	try {
	} catch (Exception e) {
	} finally {
	}
}