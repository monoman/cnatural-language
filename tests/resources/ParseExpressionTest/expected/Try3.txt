Lambda( => {
try {
}
catch (Exception e) {
}
finally {
}
}
)